Formula/gdal.rb

```diff
@@ -26,9 +26,6 @@ class Gdal < Formula
   depends_on "pkg-config" => :build
 
   depends_on "cfitsio"
-  # Work around "Symbol not found: _curl_mime_addpart"
-  # due to mismatched SDK version in Mojave.
-  depends_on "curl" if MacOS.version == :mojave
```

Comment:
Shouldn't this become a `uses_from_macos`?

Reply:
I suppose it should, yes. CI runs here take a while, though, so I'd prefer if we did this in a separate PR.

Reply:
Sure